src/components/CarbonAd/CarbonAd.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState, useRef } from 'react';
+import Script from 'react-load-script';
+
+import './CarbonAd.css';
+
+const CarbonAd = () => {
+  const [showReminder, setShowreminder] = useState(false);
+  const containerRef = useRef();
+  const verifyIsLoaded = () => {
+    const ads = document.getElementById('carbonads'); // eslint-disable-line no-undef
+    const style = window.getComputedStyle(containerRef.current); // eslint-disable-line no-undef
```

Comment:
Since **document** & **window** is undefined in this scope, we tell eslint to ignore those lines but only coz we are confident that this will work in production 😎